runtime/js-compute-runtime/builtins/request-response.cpp

```diff
@@ -120,6 +180,39 @@ bool enqueue_internal_method(JSContext *cx, JS::HandleObject receiver,
 
 } // namespace
 
+bool RequestOrResponse::process_pending_request(JSContext *cx, int32_t handle,
```

Comment:
This used to be within event_loop.cpp but I moved it into request-response as it is only used on requests